integreat_cms/integreat-cms-cli

```diff
@@ -32,7 +40,7 @@ def main():
             # pylint: disable=import-outside-toplevel,unused-import
             import django
         except ImportError:
-            raise ImportError(
+            raise ImportError from Exception(
```

Comment:
This does not make sense to me, as it _looses_ information in comparison to the previous solution. I guess what you intended was something like this?
```
        except ImportError as e:
            raise ImportError(
                "Couldn't import Django. Are you sure it's installed and "
                "available on your PYTHONPATH environment variable? Did you "
                "forget to activate a virtual environment?"
            ) from e
```
I cannot make this suggestion because it's out of reach.